.github/workflows/distribute-grain.yml

```diff
@@ -44,17 +46,13 @@ jobs:
           echo "::set-output name=pr_body::$description"
           rm grain_output.txt
 
-      - name: Create commit and PR for ledger changes
-        id: pr
-        uses: peter-evans/create-pull-request@v3
-        with:
-          branch: generated-ledger
-          branch-suffix: timestamp
-          committer: credbot <credbot@users.noreply.github.com>
-          # author appears to be overridden when the default github action
-          # token is used for checkout
-          author: credbot <credbot@users.noreply.github.com>
-          commit-message: update calculated ledger
-          title: ${{ env.PULL_REQUEST_TITLE }}
-          body: ${{ steps.pr_details.outputs.pr_body }}
-          reviewers: decentralion
+      - name: Commit ledger changes
+        run: |
+          git config user.name 'credbot'
+          git config user.email 'credbot@users.noreply.github.com'
+          git add data/ledger.json
+          git commit --allow-empty -m '${{ env.PULL_REQUEST_TITLE }}' -m '${{ steps.pr_details.outputs.pr_body }}'
+
+      - name: Push Ledger
+        run: |
+          git push
```

Comment:
should we be more specific? as in `git push origin/master`?

Reply:
The job gathers the base branch a push would be directed towards during the `checkout` step. We don't want to hard-code this in case we want to run this workflow on a grain-trigger branch for some reason (which can then later be merged as a PR if desired). Only scheduled run that target master should push to master.